package.json

```diff
@@ -6,6 +6,7 @@
     "ajv": "^6.5.0",
     "antd": "^3.4.4",
     "axios": "^0.18.0",
+    "bitcoinjs-lib": "^4.0.2",
```

Comment:
this module should be updated when bitcoingreenjs-lib is published.

Reply:
Yes